nodes/src/nodes/tool_github/IInstance.py

```
@@ -0,0 +1,1046 @@
+        stats = data.get('stats') or {}
+        result['stats'] = {'additions': stats.get('additions'), 'deletions': stats.get('deletions'), 'total': stats.get('total')}
+        result['files'] = [{'filename': f.get('filename'), 'status': f.get('status'), 'additions': f.get('additions'), 'deletions': f.get('deletions'), 'patch': f.get('patch')} for f in (data.get('files') or [])]
+        return result
+
+
+# ---------------------------------------------------------------------------
+# Module-level helpers
+# ---------------------------------------------------------------------------
+
+
+def _normalize(input_obj):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_normalize`

(ANN202)